lib/helpers/validate_dpop.js

```diff
@@ -71,8 +71,9 @@ export default async (ctx, accessToken) => {
     }
 
     {
-      const expected = new URL(ctx.oidc.urlFor(ctx.oidc.route)).href;
```

Comment:
This is needed to normalize the URL's scheme and hostname case independent, introduced 3 weeks ago in https://github.com/panva/node-oidc-provider/commit/b72d6686489bdb1a7484e1ba697614137cd36b73. You'd need to revert this change.